### DISCUSSION TITLE
Jump to window via keyboard

### DISCUSSION THREAD

**Discussion by holocronweaver** [qtile/qtile#4698] [Q&A]
In Emacs I use [ace-window](https://github.com/abo-abo/ace-window) to change focus to a specific window using only the keyboard. It works by a keybinding that changes focus in two ways:

2 windows: change focus to the other window.

more than 2 windows: show a unique character in the upper corner of each window; type a window's character to focus it.

Here is a GIF demonstrating the behavior:

![YtU5S](https://github.com/qtile/qtile/assets/1024369/718f1d3e-e280-4fd1-82cb-ae4730bbf669)

In this way with two keystrokes I can focus any window, no need for cycling focus. I have seen this feature called "jump to window" in some software.

Is there anything equivalent to `ace-window` in qtile that allows quick and exact window selection via keyboard without cycling between windows?

**Comment by jrwrigh** [comment]
No, there isn't anything that I know of. The only thing that I'm not sure how to implement for something like that would be the character in the corner of the screen. Otherwise, the rest would be pretty straight forward to implement.

**Reply by ervinpopescu** [reply]
https://qtile-extras.readthedocs.io/en/latest/manual/how_to/popup.html for the character, maybe?

**Reply by jrwrigh** [reply]
Yeah, maybe that'd work. Unfortunately, I don't really have time to try and implement it, but I'm happy to advise where I can.
